packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -45,12 +41,20 @@ const [
   restoreStateIfNeeded,
   dispatchEvent,
   runEventsInBatch,
+  actedUpdates,
 ] = ReactDOM.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.Events;
 
 function Event(suffix) {}
 
 let hasWarnedAboutDeprecatedMockComponent = false;
 
+let globalPromise = null;
```

Comment:
Explain to me again why we need to use the built-in promises? I still don't get why a thenable isn't sufficient.

Reply:
Maybe point to a test that fails if you use a thenable?

Reply:
I must have been doing something wrong. I managed to get this to work.